src/components/atoms/AnyWayButton.tsx

```diff
@@ -0,0 +1,10 @@
+import React, {FunctionComponent} from 'react';
+import {Button} from "@material-ui/core";
+
+interface IProps {
+  onClick: () => any,
+}
+
+export const AnyWayButton: FunctionComponent<IProps> = ({onClick, children}) => (
+  <Button onClick={onClick}>{children}</Button>
```

Comment:
Make sure to pass all props supported in Material-UI's button (https://material-ui.com/components/buttons/).
You can probably import the TS interface from the supported types somewhere, and then simply destrucuture props into button:
```
props => <Button {...props}>
```

We can later take specific props that you need to work with separately:
```
(customProp, ...restOfProps) => <Button customProp={myFunc(customProp)} {...restOfProps} />
```